src/test/resources/application-test.yml

```diff
@@ -40,7 +40,7 @@ hmpps.sqs:
       dlqName: cpr_delius_offender_events_queue_dlq
       subscribeTopicId: domainevents
       subscribeFilter: '{"eventType":[ "probation-case.engagement.created", "probation-case.address.created", "probation-case.address.updated", "probation-case.address.deleted","probation-case.personal-details.updated", "probation-case.alias.created","probation-case.alias.updated","probation-case.alias.deleted", "probation-case.engagement.recovered" ] }'
-      dlqMaxReceiveCount: 2
+      dlqMaxReceiveCount: 4
```

Comment:
more retries here as the faster processing means messages are processed more quickly and therefore more database conflicts arise